salt/orchestrate/edx_sandbox/build_ami.sls

```diff
@@ -25,7 +25,7 @@ build_edx_sandbox_ami_node:
     - tgt_type: compound
     - highstate: True
 
-{% set ami_name = "edx_sandbox_ami_{}".format(strftime("%Y%m%d%H%M%S")) %}
+{% set ami_name = "edx_sandbox_ami_{}".format(None | strftime("%Y%m%d%H%M%S")) %}
```

Comment:
Yeah it looks weird, but it's the way to use the strftime filter when trying to use the current time: https://docs.saltstack.com/en/latest/ref/renderers/all/salt.renderers.jinja.html#filters